include/deal.II/fe/mapping_q.h

```diff
@@ -474,43 +474,51 @@ class MappingQ : public Mapping<dim, spacedim>
      */
     QGaussLobatto<1> line_support_points;
 
+    /**
+     * A vectorized array type to reflect the necessary number of components
+     * for all interpolations to be done by this class.
```

Comment:
Could you extend this comment a bit to reflect also the aspects discussed in the PR description. 1) in the optimal case each lane is processing a component and 2) why don't we do this for 1D?